puppet/modules/quickstack/manifests/pacemaker/neutron.pp

```diff
@@ -169,7 +178,9 @@
       nova_conf_additional_params    => $nova_conf_additional_params,
       n1kv_vsm_ip                    => $n1kv_vsm_ip,
       n1kv_vsm_password              => $n1kv_vsm_password,
+      n1kv_vsm_username              => $n1kv_vsm_username,
       n1kv_plugin_additional_params  => $n1kv_plugin_additional_params,
+      n1kv_ml2_plugin_additional_params  => $n1kv_ml2_plugin_additional_params,
```

Comment:
Really minor, but could you fix the spacing here so all the '=>' line up?  Makes it easier to read